Mathlib/MeasureTheory/Constructions/BorelSpace/Metrizable.lean

```diff
@@ -25,42 +25,6 @@ variable [TopologicalSpace β] [PseudoMetrizableSpace β] [MeasurableSpace β] [
 
 open Metric
 
-/-- A limit (over a general filter) of measurable `ℝ≥0∞` valued functions is measurable. -/
-theorem measurable_of_tendsto_ennreal' {ι} {f : ι → α → ℝ≥0∞} {g : α → ℝ≥0∞} (u : Filter ι)
```

Comment:
Could you deprecate the old names, instead of erasing them completely?

Reply:
I don't know how to do that. Do you have a link to an example?

Reply:
Never mind. I found one.